London | ITP-MAY-25 | Tewodros Bekere | Module-Onboarding | Form control

iteddy16 opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

I am creating a t-shirt sale form to check their name and email, and after that, they choose their color and size and submit it. 

## Questions

Ask any questions you have for your reviewer.

iteddy16 commented:
For the form control, the first time I did it in the wireframe folder, you could see all the commit steps. After chatting with a teammate today, I know I made a mistake doing it that way, so I have created a branch for the form control and copied my work from the wireframe folder to the form control branch folder.

Amundeep-Dhaliwal commented:
Hello Tewodros, hope you are well and thank you for submitting this PR.

Was looking over this PR and had a few comments:
- Thank you for changing the title of this PR to include your full name, much appreciated
- The description of the PR has a nicely formatted checklist, is there a way that we can list the changes that we made in the change log? (example points `Have added a list of sizes for T shirt sizes` & `Have included images in the wireframe project`)
- I liked how you used multiple commits, these allow a reviewer to easily track progress in the PR. Perhaps some more descriptive comments about the changes within the commit might be good for the future.
- The form controls section seems to be in a good state, one can easily pick a T-shirt.
- Does not look like there has been much done on the wireframe side, it would be really good if you can describe what a wireframe, README & git branches are?

Overall this PR is heading in the right direction and I would encourage you to continue work.

cjyuan commented:
It looks like this PR has been reviewed. So I am removing both "Needs review" and "Review in progress" labels, and adding a "Reviewed" label to it.

iteddy16 commented:
I added more description text to the wireframe PR as suggested.

Amundeep-Dhaliwal commented:
Hello Tewodros, thank you for making your changes.

The form controls part of this project looks good, would be nice to move onto the wireframe section of the project.

iteddy16 commented:
@Amundeep-Dhaliwal Thank you for the comment. As per your suggestion, I modified the wireframe, not the form control, and the wireframe was reviewed and had a complete tag, but as per your suggestion, I added some detailed descriptions for the three questions on the wireframe. If the form control looks good and there are no other things to add to it, it's better to add a complete tag label on it because it holds me behind my schedule to complete STEP-1 of the module onboarding exercises. 
Thank you,

Amundeep-Dhaliwal commented:
Thanks for the comment Tewodros, have added the complete label.
